src/renderer/marketplace/bank-accounts/list/index.jsx

```diff
@@ -1,40 +1,53 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
 import { pricesSelectors } from 'common/prices';
 import { withStyles } from '@material-ui/core/styles';
 import { bankAccountsOperations, bankAccountsSelectors } from 'common/bank-accounts';
+/*
 import { Grid } from '@material-ui/core';
 import CircularProgress from '@material-ui/core/CircularProgress';
+*/
+import { BankingOffersPage } from './offers-page';
 
-const styles = theme => ({
-	loading: {
-		marginTop: '5em'
-	}
-});
+const styles = theme => ({});
+const MARKETPLACE_ROOT_PATH = '/main/marketplace-categories';
 
 class BankAccountsTable extends Component {
+	state = {
+		accountType: 'personal',
+		loading: false
+	};
+
 	componentDidMount() {
 		if (!this.props.bankAccounts || !this.props.bankAccounts.length) {
 			this.props.dispatch(bankAccountsOperations.loadBankAccountsOperation());
 		}
 	}
 
-	renderLoadingScreen = () => (
-		<Grid container justify="center" alignItems="center">
-			<CircularProgress size={50} className={this.props.classes.loading} />
-		</Grid>
-	);
+	onBackClick = () => this.props.dispatch(push(MARKETPLACE_ROOT_PATH));
+
+	onAccountTypeChange = accountType => this.setState({ accountType });
+
+	onDetailsClick = bank => console.log('TODO', bank);
 
 	render() {
-		// const { classes, isLoading, bank_accounts, keyRate } = this.props;
 		const { isLoading, bankAccounts, keyRate } = this.props;
-		if (isLoading) {
-			return this.renderLoadingScreen();
-		}
-		console.log(keyRate, bankAccounts);
-		// TODO: placeholder
-		return null;
+		console.log(keyRate, bankAccounts, isLoading);
```

Comment:
stray log

Reply:
Somewhere in the future, the build script will remove log calls :)